examples/bri-1/base-example/src/index.ts

```diff
@@ -188,27 +192,7 @@ export class ParticipantStack {
             this.baselineConfig?.nchainApiHost,
           );
 
-          const signerResp = (await nchain.createAccount({
-            network_id: this.baselineConfig?.networkId,
-          }));
-
-          const resp = await NChain.clientFactory(
-            this.workgroupToken,
-            this.baselineConfig?.nchainApiScheme,
-            this.baselineConfig?.nchainApiHost,
-          ).executeContract(this.contracts['verifier'].id, {
-            method: 'verify',
-            params: [[proof], ['2'], [this.baselineCircuitSetupArtifacts?.keypair!.vk]], // FIXME...
-            value: 0,
-            account_id: signerResp['id'],
-          });
-
-          console.log(resp);
-          if (!resp) {
-            return Promise.reject(`failed to verify proof: ${proof}`);
-          }
-
-          const leaf = await this.baseline?.insertLeaf(msg.sender, this.contracts['shield'].address, payload.hash);
+          const leaf = await this.baseline?.insertLeaf(msg.sender, this.contracts['shield'].address, payload.__hash);
```

Comment:
Shouldn't the `proof` + `publicInputs` + `commitmentHash` be passed to the `insertLeaf` method so that the on-chain verification can be performed and act as a gateway to allowing a new commitment to be added to the merkle-tree?